gis_processor/files_db_gis.py

```diff
@@ -66,3 +67,30 @@ def update_rel_path(
         )
         # 4: We commit the changes to the database
         self.commit()
+
+    def add_template(self, rel_path: Path, full_path: Path):
+        file = File(
```

Comment:
Du kan bruge `File.from_file` for at oprette en nye File objekt med nogle basis metadata (f.x. checksum).

Reply:
På den måde skal du kun ændre puid, signature, og action/action_data.

Reply:
Generelt kan de gøres mere simpel:



```python
file = File.from_file(full_path, root_path)
file.puid = "x-fmt/111"
file.signature = "Plain Text File"
file.action = "convert"
file.action_data = ActionData(convert=[
    ConvertAction(converter="copy", converter_type="master", outputs=["txt"]),
    ConvertAction(converter="text", converter_type="statutory", outputs=["tif"]),
])
```